i18n/locale/zh_HK.json

```diff
@@ -1,158 +1,158 @@
+    "### Model comparison\n> You can get model ID (long) from `View model information` below.\n\nCalculate a similarity between two models.": "### 模型比较\n> 模型ID(长)请于下方`查看模型信息`中获得\n\n可用于比较两模型推理相似度",
     "### Model extraction\n> Enter the path of the large file model under the 'logs' folder.\n\nThis is useful if you want to stop training halfway and manually extract and save a small model file, or if you want to test an intermediate model.": "### 模型提取\n> 输入logs文件夹下大文件模型路径\n\n适用于训一半不想训了模型没有自动提取保存小文件模型, 或者想测试中间模型的情况",
-    "### Model fusion\nCan be used to test timbre fusion.": "### Model fusion\nCan be used to test timbre fusion.",
+    "### Model fusion\nCan be used to test timbre fusion.": "### 模型融合\n可用于测试音色融合",
     "### Modify model information\n> Only supported for small model files extracted from the 'weights' folder.": "### 修改模型信息\n> 仅支持weights文件夹下提取的小模型文件",
-    "### Step 1. Fill in the experimental configuration.\nExperimental data is stored in the 'logs' folder, with each experiment having a separate folder. Manually enter the experiment name path, which contains the experimental configuration, logs, and trained model files.": "### Step 1. Fill in the experimental configuration.\nExperimental data is stored in the 'logs' folder, with each experiment having a separate folder. Manually enter the experiment name path, which contains the experimental configuration, logs, and trained model files.",
+    "### Step 1. Fill in the experimental configuration.\nExperimental data is stored in the 'logs' folder, with each experiment having a separate folder. Manually enter the experiment name path, which contains the experimental configuration, logs, and trained model files.": "### 第一步 填写实验配置\n实验数据放在logs下, 每个实验一个文件夹, 需手工输入实验名路径, 内含实验配置, 日志, 训练得到的模型文件.",
     "### Step 2. Audio processing. \n#### 1. Slicing.\nAutomatically traverse all files in the training folder that can be decoded into audio and perform slice normalization. Generates 2 wav folders in the experiment directory. Currently, only single-singer/speaker training is supported.": "### 第二步 音频处理\n#### 1. 音频切片\n自动遍历训练文件夹下所有可解码成音频的文件并进行切片归一化, 在实验目录下生成2个wav文件夹; 暂时只支持单人训练.",
     "### Step 3. Start training.\nFill in the training settings and start training the model and index.": "### 第三步 开始训练\n填写训练设置, 开始训练模型和索引.",
     "### View model information\n> Only supported for small model files extracted from the 'weights' folder.": "### 查看模型信息\n> 仅支持weights文件夹下提取的小模型文件",
     "#### 2. Feature extraction.\nUse CPU to extract pitch (if the model has pitch), use GPU to extract features (select GPU index).": "#### 2. 特征提取\n使用CPU提取音高(如果模型带音高), 使用GPU提取特征(选择卡号).",
     "Actually calculated": "实际计算",
-    "Adjust the volume envelope scaling. Closer to 0, the more it mimicks the volume of the original vocals. Can help mask noise and make volume sound more natural when set relatively low. Closer to 1 will be more of a consistently loud volume": "輸入源音量包絡替換輸出音量包絡融合比例，越靠近1越使用輸出包絡",
```

Comment:
Maybe you do not know the difference between Traditional Chinese and Simplified Chinese. You can leave the `zh_xx` files unchanged and I will look at them later.

Reply:
> Maybe you do not know the difference between Traditional Chinese and Simplified Chinese. You can leave the `zh_xx` files unchanged and I will look at them later.

Honestly i thought that zh_* files were essentially copies of each other, until i saw zh_tw

Should i revert any changes to the zh_* locales?

Reply:
Well, you should. Singapore uses Simplified Chinese so you can copy from zh_CN. The others are traditional Chinese and they are the same.